iceoryx_posh/include/iceoryx_posh/gateway/gateway_generic.hpp

```diff
@@ -34,6 +35,8 @@ namespace iox
 {
 namespace gw
 {
+using namespace iox::units::duration_literals;
```

Comment:
I thought we would like to get rid of `using namespace` in headers?

Reply:
Yes, but not in this pull request. We discussed it and said there will be another one, remember? ;-) To not let it slip through, we'll move the issue right into our sprint.